index.bs

```diff
@@ -455,6 +455,8 @@ specification.
 
 All tokens, Client, and User credentials MUST only be transmitted over TLS.
 
+All resources required to verify claims: Issuer, WebID and Client WebID; MUST only be transmitted over TLS.
```

Comment:
I think the intention here is good, but I'm concerned that this might be over-specifying things.

First, an RS may cache JWKS resources in a local keystore (hence, transfer is between local keystore and application, which may not be TLS)
Second, there may be other secure transfer mechanisms (mTLS) that are possible
Third, WebID documents (according to the WebID spec) may use either the `http` or `https` scheme
Fourth, if a WebID is a DID, the resolver may have its own mechanism to securely transfer resources

Reply:
If this is principally about WebID documents using `http` vs `https` schemes, then the text in this PR should focus on how this specification constrains how the WebID specification should be followed.